docs/src/admin-manual/configuration.md

```diff
@@ -0,0 +1,1262 @@
+# Configuration
+
+This page describes the various configuration files and settings that Enduro
+supports.
+
```

Comment:
It would be nice to mention here that the entire configuration can be set using environment variables with the `ENDURO_` prefix. E.g.: `ENDURO_API_AUTH_ENABLED`.

Reply:
ok will add. naming convention rules: 

* prefix with `ENDURO_`
* Use all uppercase
* change periods in config sections to underscores, and do the same for parameters within a section

Example - to create an environment variable of the `claimPath` parameter in the `[api.auth.oidc.abac]` config section, it would be: 

`ENDURO_API_AUTH_OIDC_ABAC_CLAIMPATH`

Correct?

Reply:
Yep, thanks!